react/features/app/actions.js

```diff
@@ -79,6 +81,7 @@ export function appNavigate(uri: ?string) {
         // FIXME: unify with web.
         if (navigator.product === 'ReactNative') {
             dispatch(disconnect());
+            navigate(screen.root);
```

Comment:
This will likely break web because we are importing native stuff.

I think it's time we move all navigation stuff to features/mobile/navigation and do the navigation in a middleware there, by looking at the SET_ROOM action.

Reply:
Tested web also and it doesn't brake. I can move forward with this.

Reply:
Wonderful!